src/Serilog.Sinks.Loggly/LoggerConfigurationLogglyExtensions.cs

```diff
@@ -59,7 +60,8 @@ public static class LoggerConfigurationLogglyExtensions
             long? bufferFileSizeLimitBytes = null,
             long? eventBodyLimitBytes = 1024 * 1024,
             LoggingLevelSwitch controlLevelSwitch = null,
-            long? retainedInvalidPayloadsLimitBytes = null)
+            long? retainedInvalidPayloadsLimitBytes = null,
+            int? retainedFileCountLimit = null)
```

Comment:
Since this is an added argument, it will be a binary-breaking change for consumers compiled against the older version. Perhaps we should version this as `5.0.0`?